### ISSUE TITLE
Make Settings Dialogue accessible with a right click on a gauge (context menu)

### ISSUE THREAD

**Issue by aviatorhh** [nohal/dashboardsk_pi#70]
**Is your feature request related to a problem? Please describe.**
It is cumbersome to go to the plugin settings via the tools/plugin window to change settings.

**Describe the solution you'd like**
When doing a right click on a gauge it should be possible to access the settings directly.

**Describe alternatives you've considered**
Implementing the feature myself 😊

**Additional context**
No touch device testing done.

**Comment by nohal** [comment]
Yes, as soon as the framework for https://github.com/nohal/dashboardsk_pi/issues/17 is in place, there should be a shortcut directly to the settings of each particular instrument.
The preferences are already directly accessible from the pager widget context menu, but I don't want to make it visible when there is only one page...
